### PR TITLE
[I18N] French translation

### PR DESCRIPTION
**Pull request by TehPeGaSuS**
As usual, some strings weren't translated due to the lacking of a proper word/meaning.

@RocketChat/android

### CODE REVIEW

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -1,194 +1,195 @@
 <resources>
     <!-- Titles -->
-    <string name="title_sign_in_your_server">Sign in your server</string>
-    <string name="title_log_in">Log in</string>
-    <string name="title_register_username">Register username</string>
-    <string name="title_sign_up">Sign up</string>
-    <string name="title_authentication">Authentication</string>
-    <string name="title_legal_terms">Legal Terms</string>
+     <string name="title_sign_in_your_server">Connectez-vous sur votre serveur</string>
```
Comment (rafaelks) [inline app/src/main/res/values-fr/strings.xml]: 👀

**Review by rafaelks — APPROVED** [review]
👏 😊

File: app/src/main/res/values-fr/strings.xml (hunk 1 above)
Comment (TehPeGaSuS) [reply app/src/main/res/values-fr/strings.xml]: yeah. French uses long sentences to say something so small. xD

Comment (rafaelks) [reply app/src/main/res/values-fr/strings.xml]: Hahah, the emoji was actually to the extra space you added 😂

Comment (TehPeGaSuS) [reply app/src/main/res/values-fr/strings.xml]: don't blame me. blame my IDE. 😂
Geany on Xubuntu 18.04 xD

**Comment by TehPeGaSuS** [comment]
huh, wait. why did the build failed?

**Comment by TehPeGaSuS** [comment]
time to sleep. if the failed checks are due to the translation let me know. I can be overlooking something.

**Comment by TehPeGaSuS** [comment]
```Errors found:
  
  /home/circleci/project/app/src/main/res/values-fr/strings.xml:56: Error: The resource string "msg_more_than_ninety_nine_unread_messages" has been marked as translatable="false" [ExtraTranslation]
      <string name="msg_more_than_ninety_nine_unread_messages">99+</string>
```
This needs a fix

**Pull request closed without merging**